docs/articles/nunit/release-notes/breaking-changes.md

```diff
@@ -130,6 +130,7 @@ Other breaking changes are grouped in the following tables.
 | NUnitLite          | NUnitLite executable tests must now reference nunit.framework in addition to NUnitLite. |
 | SetUpFixture       | Now uses `OneTimeSetUpAttribute` and `OneTimeTearDownAttribute` to designate higher-level setup and teardown methods. `SetUpAttribute` and `TearDownAttribute` are no longer allowed. |
 | TestCaseData       | The `Throws` Named Property is no longer available. Use `Assert.Throws` or `Assert.That` in your test case. |
+| TestCaseData       | The `Categories` Named Property is no longer available. Use `Properties["Category"]` instead. |
```

Comment:
Question for the reviewer:

This is the first place we're listing two breaking changes for the same listed "feature". Do we prefer to have one row per change (current) or one row per feature and then sub-delineate the affected changes on the right hand side?

Reply:
Imho:  One row per change. 

The most interesting thing is what the breaking changes are, not the action that caused them.

Reply:
I agree with @OsirisTerje 👍